packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -345,6 +354,17 @@ export class DatabaseCluster extends DatabaseClusterBase {
 
     // Get the actual subnet objects so we can depend on internet connectivity.
     const internetConnected = props.instanceProps.vpc.selectSubnets(props.instanceProps.vpcSubnets).internetConnectivityEstablished;
+
+    let monitoringRole;
+    if (props.monitoringInterval && props.monitoringInterval.toSeconds()) {
+      monitoringRole = new Role(this, "MonitoringRole", {
```

Comment:
It's nice that we default this, but can we also allow specifying this in `props`? `monitoringRole` is probably a good name for that property.

Reply:
ok, great!